README.md

```diff
@@ -2376,6 +2377,55 @@ Checks if n is between start and up to, but not including, end. If end is not sp
 
   #### Browser Support for `Math.min() and Math.max()`
 
+![Chrome][chrome-image] | ![Edge][edge-image] | ![Firefox][firefox-image] | ![IE][ie-image] | ![Opera][opera-image] | ![Safari][safari-image]
+:-: | :-: | :-: | :-: | :-: | :-: |
+✔  |  ✔  |  ✔  |  ✔  |  ✔  |  ✔  |
+
+**[⬆ back to top](#quick-links)**
+
+  ### _.random
+Produces a random number between the inclusive lower and upper bounds. If only one argument is provided a number between 0 and the given number is returned. If floating is true, or either lower or upper are floats, a floating-point number is returned instead of an integer.
+
+  ```js
+    // Lodash
+    _.random(0, 5);
+    // => an integer between 0 and 5
+    
+    _.random(5);
+    // => also an integer between 0 and 5
+    
+    _.random(5, true);
+    // => a floating-point number between 0 and 5
+    
+    _.random(1.2, 5.2);
+    // => a floating-point number between 1.2 and 5.2
+
+    //Native
+    const random = (lower, upper) => {
+      if(!upper || typeof upper === 'boolean') {
+        upper = lower;
+        lower = 0;
+      }
+      
+      let randomic = Math.random() * upper;
+      return randomic >= lower ? randomic : random(lower, upper);
+    }
```

Comment:
Better:

''''js
const random = (lower, upper = 0) => lower + Math.random() * (upper - lower);
'''

Also does lodash specifically ask that the lower bound be the first parameter? Because if lower is actually higher than upper this does random stuff, if so you should replace the variable names with a and b, and lower with Math.min([a, b]) etc.

Reply:
Hi,

The lodash don't specifically that the bound be the first paramter but thats sounds good. Analysing a semantic implementation, the correct way of use `_.random` method, is pass the lower value into first argument, then upper.

I used the lower and upper nomenclature, because of lodash's oficial documentation, case doesn't have the rules for paramters orders, i'm according with 'a' and 'b' instead 'lower' and 'upper'.

Btw, follow the link: [https://lodash.com/docs/4.17.11#random](https://lodash.com/docs/4.17.11#random)

Thank's for you proposal and analysing!

Reply:
There was an error in my above implementation. Also, this can do everything lodash can do (probably less compatibility):

```js
// a%b is rest when a divided by b, so a % 1 = 0 when `a` a integer, otherwise =/= 0 and thus converted to true by the or (||)
const random = (lower, upper = 0, floating = lower % 1 || upper % 1) => {
    const rdm = lower + Math.random() * Math.abs(upper - lower);
    return floating ? rdm : Math.round(rdm)
}
```

Reply:
clamp:

```js
const clamp = (number, ...bounds) => {
    lower = Math.min(bounds);
    higher = Math.max(bounds);
    if (number < lower) {return lower;} 
    if (number > higher) {return higher;}
    return number;
}
```